### ISSUE TITLE
Update man

### ISSUE THREAD

**Issue by sachaz** [gnebbia/kb#101]
Hi,
The upgrade to version 0.1.7 on Linux Arch changes the kb database directory. Before for me it was  in ~/.kb now in .local/share/kb/ 
The upgrade doesn't migrate the previous base to the new path and there is no information about kb files in the help or where are they stored.
Could be useful to have a migration on the upgrades, documentations on files structure and where they are.

Anyway thanks for this nice tool <3

**Comment by gagenon** [comment]
Can confirm this issue in the AUR version of KB 0.1.7. Additionally, importing a 'kb export' does not work in this scenario. (I export my KB nightly with a cron job.) Copying the files from ~/.kb to ~/.local/share/kb did restore my KB, however.

**Comment by gnebbia** [comment]
Yes we are sorry for that, we should have notified users. We wanted to comply with other GNU/Linux applications which suggest to put data in that directory.
We will write a note in the documentation,
Thanks for your feedback!
